__test__/integration/adapters/collateralized_simple_interest_loan_adapter/runners/index.ts

```diff
@@ -0,0 +1,3 @@
+import { ReturnCollateralRunner } from "./return_collateral_runner";
+
+export { ReturnCollateralRunner };
```

Comment:
Nit: there's a cleaner way of doing this I learned recently:

`export { ReturnCollateralRunner } from "./return_collateral_runner";`